holidays/countries/pakistan.py

```diff
@@ -89,6 +99,28 @@ def _populate_public_holidays(self):
         self._add_ashura_eve(name)
         self._add_ashura_day(name)
 
+        self._populate_subdiv_holidays()
```

Comment:
This is no longer added in in newer implementations.

Instead, convert them to `_populate_subdiv_[subdiv_code]_public_holidays` like in [macau.py](https://github.com/vacanza/holidays/blob/dev/holidays/countries/macau.py#L381)